apollo-federation/src/schema/validators/cache_tag.rs

```diff
@@ -432,18 +439,12 @@ enum CacheTagValidationError {
         "Object \"{0}\" is not an entity. cacheTag can only apply on resolvable entities, object containing at least 1 @key directive and resolvable"
     )]
     CacheTagEntityNotResolvable(Name),
-    #[error(
-        "Unknown arguments used with $args in cacheTag format \"{format}\" on field \"{field_name}\" for type \"{type_name}\""
-    )]
-    CacheTagFormatArgumentUnknown {
-        type_name: Name,
-        field_name: Name,
-        format: String,
-    },
     #[error(
         "Each entity field referenced in a @cacheTag format (applied on entity type) must be a member of every @key field set. In other words, when there are multiple @key fields on the type, the referenced field(s) must be limited to their intersection. Bad cacheTag format \"{format}\" on type \"{type_name}\""
     )]
     CacheTagInvalidFormatFieldSetOnEntity { type_name: Name, format: String },
+    #[error("@cacheTag can only use non nullable field but \"{field_name}\" in format is nullable")]
```

Comment:
Just a English thing:

```suggestion
    #[error("@cacheTag format references a nullable field \"{field_name}\"")]
```